### PR TITLE
[WIP] Add "project_code" field to Build model

### PR DESCRIPTION
**Pull request by SchrodingersGat**
(cherry picked from commit 0a1147b71d99c515e4b721e5a81708f7c70625a6)

Minimum working example to see what errors are thrown

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair getting the same series of errors with a very minimum working example

File: InvenTree/build/models.py [hunk 1]
```diff
@@ -31,9 +31,10 @@
 import InvenTree.ready
 import InvenTree.tasks
 
+from common.models import ProjectCode
+from common.notifications import trigger_notification
```
Comment (matmair) [inline InvenTree/build/models.py:35]: This line seems to be the issue. Reverting this made it run locally.

File: InvenTree/build/models.py [hunk 2]
```diff
@@ -546,7 +555,7 @@ def complete_build(self, user):
             }
         }
 
-        common.notifications.trigger_notification(
+        trigger_notification(
```
Comment (matmair) [inline InvenTree/build/models.py:558]: This needs to be changed too

**Review by matmair — COMMENTED** [review]
I think I found it @SchrodingersGat

**Comment by SchrodingersGat** [comment]
@matmair some interesting updates now. Only the migration tests are failing

**Comment by matmair** [comment]
@SchrodingersGat can you reproduce it not working on your device locally? I seem to be unable

**Comment by SchrodingersGat** [comment]
I don't get the same errors running locally

**Comment by wolflu05** [comment]
My pr didn't help right?

**Comment by SchrodingersGat** [comment]
> My pr didn't help right?

Unfortunately not. I'm really stuck on this one!

**Pull request closed without merging**